lnrpc/rpc.proto

```diff
@@ -1483,6 +1483,13 @@ message ListChannelsResponse {
     repeated Channel channels = 11;
 }
 
+enum Initiator {
```

Comment:
Imho less generic name may work better. Eg. `ChannelInitiator`

Reply:
We also use this enum for close initiators, so I'm not sure that fits? `ChannelInitiator` sounds like just opening to me?